packages/@aws-cdk/aws-route53-targets/lib/classic-load-balancer-target.ts

```diff
@@ -0,0 +1,17 @@
+import elb = require('@aws-cdk/aws-elasticloadbalancing');
+import route53 = require('@aws-cdk/aws-route53');
+
+/**
+ * Use a classic ELB as an alias record target
+ */
+export class ClassicLoadBalancerTarget implements route53.IAliasRecordTarget {
+  constructor(private readonly loadBalancer: elb.LoadBalancer) {
```

Comment:
I ended up creating a separate class, please tell me if you'd prefer having an union. The only issue is that the `hotsedZoneId` property is mapped differently in v1 versus v2,`loadBalancerCanonicalHostedZoneNameId`  and `loadBalancerCanonicalHostedZoneId` respectively